app/src/shared/components/ImageUpload/index.jsx

```diff
@@ -111,7 +111,7 @@ const ImageUpload = ({
                 <PngIcon
                     className={styles.icon}
                     name="imageUpload"
-                    alt={I18n.t('imageUpload.coverImage.upload')}
+                    alt=""
```

Comment:
Unrelated, avoid `<br />` tag showing in `alt`, any there is a similar text below the image